lib/Backend/GlobOpt.cpp

```diff
@@ -1060,6 +1060,9 @@ GlobOpt::MergePredBlocksValueMaps(BasicBlock *block)
         this->KillAllObjectTypes();
     }
 
+    this->blockData.capturedArgs = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
+    this->blockData.changedSyms = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
```

Comment:
Oh, I though you already had code in CloneBlockData but it is in CopyBlockData.
What I mean is that you should put this in InitBlockData/CloneBlockData/ReuseBlockData like all the other data structures.  This will allow better reuse.

---
In reply to: [68686479](https://github.com/Microsoft/ChakraCore/pull/1152#discussion_r68686479) [](ancestors = 68686479,68328885,68328456)